README.md

```diff
@@ -0,0 +1,33 @@
+# ViaRE's Reference
+
+This is the place in where we share the list of references we use to keep developing
+[ViaRE](https://github.com/daque-dev/viare).
+
+## Content
+- [The proccess](#the-process) (how to add more references to this repo)
+
+## The proccess
+
+Every reference we find must go through this proccess:
+
+- **The Finding**: We find a book, website, article, magazine, etc. that we consider
+may be useful to keep generating ideas, getting context, or keeping us interested.
+When we skim the table of contents or something similar, we should take note of what
+the book could give to ViaRe, and then the book makes it to the `finding list`.
+
+- **Note-taking**: When any member of [Daque](https://github.com/orgs/daque-dev/people)
+starts to read an item inside `finding list`, he/she must start to take notes on that
```

Comment:
Referring as a member/contributor as "he/she" seems kinda forced. What is a more general noun? Does 'it' fits?

Reply:
If I remember correctly, you can use "them".

Reply:
Or rather, "they", in this context.

Reply:
Ok. Fixed in last commit.